ui/src/components/logs/ContainerLogsView.js

```diff
@@ -0,0 +1,211 @@
+import React, {
+  Fragment,
+  useContext,
+  useEffect,
+  useMemo,
+  useState
+} from "react";
+import {
+  EuiEmptyPrompt,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLoadingContent,
+  EuiPageContent,
+  EuiSpacer,
+  EuiTextColor,
+  EuiTitle
+} from "@elastic/eui";
+import { LazyLog, ScrollFollow } from "react-lazylog";
+import { AuthContext } from "@gojek/mlp-ui";
+import config from "../../config";
+import mocks from "../../mocks";
+import { LogsSearchBar } from "./LogsSearchBar";
+import { useMerlinApi } from "../../hooks/useMerlinApi";
+import StackdriverLink from "./StackdriverLink";
+import { createStackdriverUrl } from "../../utils/createStackdriverUrl";
+
+const querystring = require("querystring");
+
+const componentOrder = [
+  "image_builder",
+  "model",
+  "transformer",
+  "batch_job_driver",
+  "batch_job_executor"
+];
+
+export const ContainerLogsView = ({
+  projectId,
+  model,
+  versionId,
+  jobId,
+  fetchContainerURL
+}) => {
+  const [{ data: project, isLoaded: projectLoaded }] = useMerlinApi(
+    `/projects/${projectId}`,
+    { mock: mocks.project },
+    []
+  );
+
+  const [params, setParams] = useState({
+    component_type: "",
+    tail_lines: "1000"
+  });
+
+  const [{ data: containers, isLoaded: containersLoaded }] = useMerlinApi(
+    fetchContainerURL,
+    { mock: mocks.containerOptions },
+    []
+  );
+
+  useEffect(
+    () => {
+      if (containersLoaded) {
```

Comment:
Can it be triggered periodically using timer? I think currently user need to refresh the page in order to update the list of containers. This will be a problem of discoverability.

Reply:
It's complex to achieve that. Updating the `containers` list periodically would eventually update the LazyLog's Stream URL. However, any change in LazyLog's Stream URL would re-render the whole component and re-stream the API instead of appending the new Stream API log. It would need a whole rework of our Logging logic.

Reply:
Why do we need to update the LazyLog's Stream URL? The intention is just to always provide most up to date list of list available pod. 

<img width="193" alt="Screenshot 2021-07-29 at 3 42 48 PM" src="https://user-images.githubusercontent.com/4023015/127451987-e095e455-9e6a-4a36-a980-426a962e2989.png">

Otherwise, we can always shows all possible choice, but return no logs if the associated components doesn't have any container.

Reply:
Previously, I thought it would also update the request to log's stream URL because it was coupled. I have updated the UI logic to separately update the component list and stream the logs.